lib/rouge/demos/perl6pod

```diff
@@ -0,0 +1,123 @@
+#! /usr/bin/env perl6
```

Comment:
The "demos" file is meant to give a very brief visual example. E.g. like the text boxes here: http://rouge.jneen.net/
Please cut it down to something shorter. Most are between 10-30 lines.